pkg/cmd/cloudscale.go

```diff
@@ -123,7 +123,7 @@ func CloudscaleCmds(allMetrics map[string]map[string]prometheus.Counter) *cli.Co
 			wg.Add(1)
 			go func() {
 				for {
-					if time.Now().Hour() >= billingHour {
```

Comment:
I guess this is only for debugging? Please make sure you revert this again before merge.